department-of-veterans-affairs/vets-website
# Burial Benefits - General Cleanup

## Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
General cleanup on the Burial Benefits form (21P-530EZ) including:
- Use web component patterns where applicable (including schema)
- Remove extraneous spread operators and html
- Remove text utility classes and use inherited styles
- Ensure all external links include (opens in new tab) text

## Issue
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/95706

## Related issue(s)
- [Audit Documentation](https://docs.google.com/document/d/1Fd833bAuIeNfj4LGQn_J1A67W9CcGELgvfVlR9Th5dU/edit?usp=sharing)

## Associated Pull Request(s)
n/a

## How to run in local environment
1. Check out this branch locally
2. Run `vets-website` and `vets-api`
3. In your local environment, set the `burial_form_enabled ` flipper to 'enabled' at http://localhost:3000/flipper/features/burial_form_enabled
4. Go to `http://localhost:3001/burials-memorials/veterans-burial-allowance/apply-for-allowance-form-21p-530ez/` in your browser

## How to verify
1. Sign in and start the application
2. Verify that the application can be saved in progress
3. Verify that the application can be resumed
4. Verify that the application can be submitted

## What areas of the site does it impact?
Burial Benefits Application

## Screenshots
n/a

### Quality Assurance & Testing

### Error Handling

### Authentication

## Requested Feedback
(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
